docs/articles/documentation/test-api/selecting-page-elements/selectors/selector-options.md

```diff
@@ -98,7 +98,7 @@ This option is in effect when TestCafe waits for the selector to return a page e
 If the target element is not visible, the selector throws an exception in all these cases.
 
 Note that when a selector is passed to a [test action](../../actions/README.md) as an identifier for the target element,
```

Comment:
Note that when a selector is passed to a [test action](../../actions/README.md) as the target element's identifier, the target element should be visible regardless of the `visibilityCheck` option.